TensorIO/Classes/TensorFlow/TIOTensorFlowModel/TIOTensorFlowModel.mm

```diff
@@ -20,11 +20,16 @@
 
 //  TODO: Using class string to identity tensorflow model in model.json, identify some other way
 //  TODO: Overloading model.file in model.json to point to predict directory, must also point to train and eval dirs
+//  TODO: Duplicating input/output parsing but may need backend specific parsing as well
+//  TODO: Duplicated TensorType defines, should be defined elsewhere
```

Comment:
Untracked, not sure if this is really that important or if I'm still using these elsewhere